pkg/controller/config/config_controller.go

```diff
@@ -150,13 +151,25 @@ func (r *ReconcileConfig) Reconcile(ctx context.Context, request reconcile.Reque
 	}
 	exists := true
 	instance := &configv1alpha1.Config{}
+	cfgTracker := r.getTracker()
+
 	err := r.reader.Get(ctx, request.NamespacedName, instance)
 	if err != nil {
+		// use a tempInstance for cfg tracker
```

Comment:
I assume you're making this b/c it's required for TryCancelExpect.  In that case i'd just put it next to that call.